torchtune/utils/metric_logging.py

```diff
@@ -147,9 +160,10 @@ class WandBLogger(MetricLoggerInterface):
 
     def __init__(
         self,
-        project: str,
+        project: str = "torchtune",
         entity: Optional[str] = None,
         group: Optional[str] = None,
+        log_strategy: Optional[str] = "main",
```

Comment:
nit: could type as a literal, like we do e.g. [here](https://github.com/pytorch/torchtune/blob/e7f1a6d8e7b7611a2528e97e594242453d8ffd86/torchtune/data/_types.py#L10)